xcodeproj/internal/xcodeprojinfo.bzl

```
@@ -263,12 +268,8 @@ def _skip_target(
                     id = info.xcode_target.id,
                     automatic_target_info = automatic_target_info,
                 )
-                for attr, info in transitive_infos
-                if (target and
-                    attr in deps_attrs and
-                    info.xcode_target and
-                    automatic_target_info.args)
-            ],
+                for info in deps_transitive_infos
+            ] if automatic_target_info.args else None,
```

Guess it doesn't change the behavior so it's not necessary/never `None`?
```suggestion
            ] if target and automatic_target_info.args else None,
```

---

There was a time when `target` could be `None`, but now it's called only from the aspect, so it's always non-`None`.